dataland-frontend/tests/e2e/fixtures/lksg/LksgDataFixtures.ts

```diff
@@ -21,14 +21,38 @@ export function generateProductionSite(): ProductionSite {
       InHouseProductionOrContractProcessing.InHouseProduction,
       InHouseProductionOrContractProcessing.ContractProcessing,
     ]),
-    country: randomStringOrUndefined(faker.address.country()),
+    country: randomStringOrUndefined(faker.address.countryCode()),
     city: randomStringOrUndefined(faker.address.city()),
     streetAndHouseNumber: randomStringOrUndefined(faker.address.street() + " " + faker.address.buildingNumber()),
     postalCode: randomStringOrUndefined(faker.address.zipCode()),
     listOfGoodsOrServices: fakeGoodsOrServices,
   };
 }
 
+/**
+ * Generates a random production site without any undefined values
```

Comment:
Why do we need that? Do we have a test that relies on all fields beeing populated?

Reply:
If that is indeed the case, how about introducing the fraction of undefined values as an optional parameter that is by default set to the constant value of expected undefined but can also be reduced to 0.

Reply:
This prepared fixture is used in the EditDataset.ts test.
I think that the author wanted to ensure, that prefilling production sites works if you click on the EDIT button for an lksg dataset.
Therefore he needed an lksg dataset which has production sites.

Is this ok?

Reply:
@MarcTM01  can you confirm?

Reply:
I like that idea. Implemented (We need this test for the edit-dataset test as the upload form validates that all fields are specified for production sites)